opus_load.py

```diff
@@ -0,0 +1,14 @@
+import discord
+
+opus_libs = ['libopus.0.dylib', 'libopus-0.dll', 'libopus.so.0']
+# must download libopus for your respective OS for bot to use voice
+	# mac and windows opus libs added, but linux needs to do this step
+	# either place in this directory set up path to make it recognizeable
```

Comment:
Hmmmmmm... Bundling libraries (e.g. including [this](https://github.com/BunnyApocalypse/tempbot/pull/5/files#diff-fc87716e506b5bb44162748810bf3c06)) is something we probably don't want to do going into the future, but I'll approve it for now...